fischkapp/src/App.tsx

```diff
@@ -4,12 +4,25 @@ import { theme } from "./components/styles/theme/theme";
 import { Wrapper } from "./components/styles/Wrapper";
 import { Header } from "./components/Header";
 import { GlobalStyle } from "./components/styles/GlobalStyles";
+import { FrontCard } from "./components/FrontCard";
+import { BackCard } from "./components/BackCard";
+import { useState } from "react";
+
 function App() {
+  const [cardStatus, setCardStatus] = useState<boolean>(true);
+
+  const getCardStatus = (status: boolean) => {
+    setCardStatus(status);
+    console.log(status);
+  };
+
   return (
     <ThemeProvider theme={theme}>
       <GlobalStyle />
       <Wrapper>
         <Header />
+        <FrontCard getCardStatus={getCardStatus} cardStatus={cardStatus} />
```

Comment:
**nitpick:** I think that it should be a single component where you toggle between `FrontCard`, and `BackCard`. Currently it will be very hard to display more cards in a list